### PR TITLE
Ft-server-get-center-details-152856013

### PR DESCRIPTION
**Pull request by nzediegwu1**
#### What does this PR do?
GIVEN a User
WHEN User sends a request to route GET: /centers/<centerId>,
THEN upon success User should receive a success response containing the details of a center or an error message upon failure

#### Description of Task to be completed?
create GET: /centers/<centerId> route in centersRouter
define getCenterDetails action in centersController

#### How should this be manually tested?
using postman

#### Any background context you want to provide?
none

#### What are the relevant pivotal tracker stories? (If applicable)
https://www.pivotaltracker.com/story/show/152856013

#### Screenshots (If applicable)
none

### CODE REVIEW

**Comment by oreoluwade** [comment]
The "what does this PR do" section does not need to be that elaborate. Could simply be `Create an endpoint to get center details`.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,4 +18,4 @@
```
Comment (oreoluwade) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: How come this file is among the files changed in this branch/pull request? Did you add an additional line?

Comment (nzediegwu1) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: Each time I created a new branch, I had to add or remove new line in other to have changes that would enable me to commit the file into my new branch

**Comment by nzediegwu1** [comment]
Well noted

**Pull request merged**